app/new-project/page.tsx

```diff
@@ -294,11 +353,27 @@ function NewProjectForm() {
     },
   });
 
+  const [filesSelected, setFilesSelected] = useState<FileList | null>(null);
```

Comment:
might be simpler to not use null and set it to `new FileList` instead so the `null` case doensn't need to be handled separately